AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/AWSCognitoAuthPlugin+Configure.swift

```diff
@@ -110,6 +120,11 @@ extension AWSCognitoAuthPlugin {
                 frameworkMetadata: AmplifyAWSServiceConfiguration.frameworkMetaData(),
                 region: identityPoolConfig.region
             )
+
+            #if os(watchOS) || os(tvOS)
+            configuration.httpClientEngine = FoundationClientEngine()
```

Comment:
I now wonder if we should be using the `httpClientEngineProxy` here as well, but that's a story for another time.